js/ui/html_editor/modules/toolbar.js

```diff
@@ -181,6 +189,41 @@ if(Quill) {
             }
         }
 
+        _subscribeFormatHotKeys() {
+            this.quill.keyboard.addBinding({
+                which: 66,
```

Comment:
Don't use "magic numbers"

```js
const KEY_CODES = {
  b: 66
};
```

```suggestion
                which: KEY_CODES.B,
```